### ISSUE TITLE
Rawhide build broken on clap_lex

### ISSUE THREAD

**Issue by jw3** [ctc-oss/fapolicy-analyzer#722]
**tl;dr;** There is no longer a `clap_lex` 0.2.x package available for fc38, which breaks Clap 3.2.x.  Which breaks our fc38 build.

`rust-clap_lex` was upgraded to 0.3.0 on Dec 24. 

https://marc.info/?l=fedora-devel-list&m=167201078230123&w=2

```
Package:      rust-clap_lex-0.3.0-1.fc38
Old package:  rust-clap_lex-0.2.4-3.fc37
Summary:      Minimal, flexible command line parser
RPMs:         rust-clap_lex+default-devel rust-clap_lex-devel
Size:         24.80 KiB
Size change:  145 B
Changelog:
  * Sat Dec 24 2022 Laiot <carmelo.sarta.main@gmail.com> 0.3.0-1
  - Update to 0.3.0
```

The current FC38 version of Clap `3.2.20` requires clap_lex `0.2.2`, as seen in the Cargo.toml

https://github.com/clap-rs/clap/blob/v3.2.20/Cargo.toml#L91

it also is seen in the RPM dependencies

```
Error: 
 Problem: conflicting requests
  - nothing provides (crate(clap_lex/default) >= 0.2.2 with crate(clap_lex/default) < 0.3.0~) needed by rust-clap-devel-3.2.20-1.fc38.noarch
```

I would expect that the FC package upgrade would have caught this in a test somewhere, so perhaps I am missing something. :thinking:  

The el and fc37 builds continue to work.

The fc38 build is submitted for the Fedora review, so this is blocking #715

**Comment by jw3** [comment]
Avoiding for now by removing the tools from the rpm build, which needed done anyways.
